tox.ini

```diff
@@ -1,7 +1,9 @@
 [tox]
 envlist = py27, py35
```

Comment:
Well you know this is just compatibility testing right? So testing against the latest version of python will make sure we support it as well. So basically we are saying that this project can run from 2.7 to 3.5... Including 3.4 (unless there are some hickups in 3.4 that I missed and are not forward or backward compatible)